test/multipart-security.test.js

```diff
@@ -288,28 +293,28 @@ test('should use default for fileSize', async function (t) {
 
   try {
     const [res] = await once(req, 'response')
-    t.equal(res.statusCode, 413, 'status code equal')
+    t.assert.strictEqual(res.statusCode, 413, 'status code equal')
     res.resume()
     await once(res, 'end')
   } catch (error) {
-    t.error(error, 'request')
+    t.assert.ifError(error)
   }
 })
 
-test('should use default for parts - 1000', function (t) {
+test('should use default for parts - 1000', function (t, done) {
   const fastify = Fastify()
-  t.teardown(fastify.close.bind(fastify))
+  t.after(() => fastify.close())
 
   fastify.register(multipart)
 
   fastify.post('/', async function (req, reply) {
     try {
       // eslint-disable-next-lint no-empty
       for await (const _ of req.parts()) { console.assert(_) }
-      t.fail('should throw on 1001')
+      t.assert.ok('should throw on 1001')
```

Comment:
@matteo-gobbo 

Just to clarify, `t.assert.ok` isn’t the same as `t.fail`.  
`assert` also provides a `fail` method, which might be helpful here:  

https://nodejs.org/api/assert.html#assertfailmessage